.github/workflows/_deploy_production.yml

```diff
@@ -58,6 +58,39 @@ jobs:
           docker buildx imagetools create \
             -t ${{ steps.login-production.outputs.registry }}/firezone/${{ matrix.image }}:${{ inputs.tag }} \
             $SOURCE_TAG
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@v2
+        with:
+          workload_identity_provider: "projects/397012414171/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions"
+          service_account: "github-actions@github-iam-387915.iam.gserviceaccount.com"
+          export_environment_variables: true
+          create_credentials_file: true
+      - name: Copy Google Cloud Storage binaries to "edge" version
+        # TODO: Add relay here when we deploy Relay from prod artifacts instead of Docker
```

Comment:
Let's add them right away, doesn't cost for us but somebody will not know about this comment and will be looking why they are not published :)

Reply:
I would normally agree, but I might suggest we hold because, to do that properly we need to:

- Update `publish.yml` to publish versioned Relays too (and start versioning Relay changes)
- Add `arm64` and `armv7l` architectures to the Relay builds (we only build for amd64 currently because that's all we need to)

Reply:
Let's put this comment to the TODO then so this knowledge is not lost and LGTM.